hooks/post/30-upgrade.rb

```diff
@@ -1,3 +1,7 @@
+require 'fileutils'
+
+STEP_DIRECTORY='/etc/foreman-installer/hooks/post/'
```

Comment:
I guess it's fine since you do mkpath, but I think I'd prefer if it was a more obvious name